cpp/OcvPersonDetection/plugin-files/descriptor/descriptor.json

```diff
@@ -60,11 +60,23 @@
         },
         {
           "name": "MERGE_TRACKS",
-          "description": "When set to true, the detection response handler will merge tracks spanning segment boundaries.",
+          "description": "In the context of videos, when set to true, the detection response handler will merge tracks spanning segment boundaries.",
           "type": "BOOLEAN",
           "defaultValue": "false"
         },
         {
+          "name": "MIN_GAP_BETWEEN_TRACKS",
+          "description": "In the context of videos, defines the minimum gap in frames to preserve between similar tracks. Similar tracks with fewer than the allowed number of frames between them will be merged into a single track. If MERGE_TRACKS is false, this has no effect.",
+          "type": "INT",
+          "defaultValue": "1"
+        },
+        {
+          "name": "MIN_TRACK_LENGTH",
+          "description": "In the context of videos, defines the minimum length in frames for tracks, which must be met or exceeded. Tracks shorter than this minimum length will be silently discarded.",
```

Comment:
Let's simplify:

"In the context of videos, defines the track length in frames. Tracks shorter than this minimum length will be silently discarded.